atomic_reactor/plugins/pre_koji_parent.py

```diff
@@ -79,6 +80,7 @@ def __init__(self, tasker, workflow, koji_hub=None, koji_ssl_certs_dir=None,
         self._base_image_build = None
         self._parent_builds = {}
         self._poll_start = None
+        self._registries = get_registries(workflow, {})
```

Comment:
use get_source_registry instead like in pull_base_image

Reply:
Done